core-bundle/assets/scripts/core.js

```diff
@@ -2291,7 +2295,7 @@ window.addEvent('domready', function() {
 	Backend.enableToggleSelect();
 
 	Theme.stopClickPropagation();
-	Theme.setupTextareaResizing();
+	//Theme.setupTextareaResizing();
```

Comment:
I commented these out rather than removing them so it's easier to merge the other 🐮 PRs (conflicts)